ci/do_ci.sh

```diff
@@ -321,14 +321,12 @@ elif [[ "$1" == "bazel.nortti" ]]; then
 elif [[ "$1" == "bazel.asan" ]]; then
   bazel $BAZEL_STARTUP_OPTIONS test --config=asan $BAZEL_TEST_OPTIONS_ASYNC //...
   bazel $BAZEL_STARTUP_OPTIONS run --config=asan $BAZEL_TEST_OPTIONS_ASYNC \
-  //examples/metrics_simple:metrics_ostream_example > /dev/null
```

Comment:
Only half the command is removed.

Delete this line as well ?

```
bazel $BAZEL_STARTUP_OPTIONS run --config=asan $BAZEL_TEST_OPTIONS_ASYNC \
```

Reply:
Yes, thanks for the catch.

Reply:
Good catch.